web_app/tabs/i3_mno.py

```diff
@@ -1,84 +1,19 @@
 """Conditional in-cell drop-down menu with IceCube WBS MoU info."""
 
-from collections import OrderedDict
 from typing import Dict, List, Tuple, Union
 
-import dash  # type: ignore[import]
-
-# --------------------------------------------------------------------------------------
-# Layout
 import dash_bootstrap_components as dbc  # type: ignore[import]
 import dash_core_components as dcc  # type: ignore[import]
 import dash_html_components as html  # type: ignore[import]
 import dash_table as dt  # type: ignore[import]
-import pandas as pd  # type: ignore[import]
 from dash.dependencies import Input, Output, State  # type: ignore[import]
 
 from ..config import app
-from ..utils.styles import (
-    CENTERED_100,
-    HIDDEN,
-    SHORT_HR,
-    WIDTH_22_5,
-    WIDTH_30,
-    WIDTH_45,
-)
+from ..utils import data
+from ..utils.styles import CENTERED_100, WIDTH_45
```

Comment:
Are we OK with relative imports?  I generally am and I don't think we have clear and set standards, but some people are opposed.

Reply:
I am. My reasoning is that `web_app` is a module, so relative imports are to be expected.